commit 740d337e89320dbd1ba5c011333193cbb5eb952a
Author: John Marshall <jmarshall@hey.com>
Date:   Sun Sep 11 17:45:11 2022 +0100

    Import ABCs from collections.abc (Python 3.10 compatibility)

    These classes are no longer available in collections in 3.10.
    The collections.abc module was introduced in 3.3, and atropos
    supports only Python >= 3.3, so fortunately there is no need
    to make this conditional.

diff --git a/atropos/commands/base.py b/atropos/commands/base.py
index 2a05b6c..0a5cd9e 100644
--- a/atropos/commands/base.py
+++ b/atropos/commands/base.py
@@ -1,6 +1,6 @@
 """Common classes/functions used in commands.
 """
-from collections import Sequence
+from collections.abc import Sequence
 import copy
 import platform
 import sys
diff --git a/atropos/commands/trim/__init__.py b/atropos/commands/trim/__init__.py
index c8d6cb8..536c2a5 100644
--- a/atropos/commands/trim/__init__.py
+++ b/atropos/commands/trim/__init__.py
@@ -1,6 +1,7 @@
 """Implementation of the 'trim' command.
 """
-from collections import Sequence, defaultdict
+from collections import defaultdict
+from collections.abc import Sequence
 import logging
 import os
 import sys
diff --git a/atropos/util/__init__.py b/atropos/util/__init__.py
index d5d9bf3..68af7a2 100644
--- a/atropos/util/__init__.py
+++ b/atropos/util/__init__.py
@@ -1,6 +1,7 @@
 """Widely useful utility methods.
 """
-from collections import OrderedDict, Iterable, Sequence
+from collections import OrderedDict
+from collections.abc import Iterable, Sequence
 from datetime import datetime
 import errno
 import functools
